typings/react-dropzone.d.ts

```diff
@@ -66,8 +68,8 @@ export type DropzoneState = DropzoneRef & {
   isDragAccept: boolean;
   isDragReject: boolean;
   isFileDialogActive: boolean;
-  draggedFiles: File[];
-  acceptedFiles: File[];
+  draggedFiles: FileWithPath[];
+  acceptedFiles: FileWithPath[];
```

Comment:
I've explained why you cannot do this. But we could use generics the same way we do for `onDrop`/`onDropAccepted`.

Reply:
The variable T cannot be used because here is a concrete value and not a function
I changed the way of writing, hope you like it

Reply:
<img width="805" alt="Screen Shot 2022-04-25 at 14 37 59" src="https://user-images.githubusercontent.com/54422571/165033724-3e5d5e6f-ac63-45d6-b83a-6c06fddce2ba.png">